### ISSUE TITLE
Strange behaviour of the routes and asset paths broken

### ISSUE THREAD

**Issue by shinesoftware** [basz/SlmLocale#67]
First of all thanks for your module jurians!
## First Configuration

I have implemented it by this simple configuration:

```
'slm_locale' => array(
    'default' => 'en-US',
    'supported' => array('en-US','it-IT','es-419'),
    'strategies' => array(
                        array(
                            'name' => 'uripath',
                            'options' => array(
                                'redirect_when_found' => true,
                                'redirect_to_canonical' => false,
                                'aliases' => array('en' => 'en-US','es' => 'es-ES','es' => 'es-419','it' => 'it-IT'),
                                ),
                            ),
                            'acceptlanguage',
                        ),
),
```

Then when I try to call the homepage http://www.mysite.com/ I get a 404 for the assets:

```
"NetworkError: 404 Not Found - http://www.site.com/it/css/homepage.css"
"NetworkError: 404 Not Found - http://www.site.com/it/js/application.js"
"NetworkError: 404 Not Found - http://www.site.com/it/css/application.css"
"NetworkError: 404 Not Found - http://www.site.com/it/js/application.js"
```
## Second Configuration

With this new configuration I get another strange behaviour: http://www.site.com/ returns a 404 error but the path of the assets is correct.

```
'slm_locale' => array(
    'default' => 'en-US',
    'supported' => array('en-US','it-IT','es-419'),
    'strategies' => array(
                        array(
                            'name' => 'uripath',
                            'options' => array(
                                'redirect_when_found' => false,
                                'redirect_to_canonical' => false,
                                'aliases' => array('en' => 'en-US','es' => 'es-ES','es' => 'es-419','it' => 'it-IT'),
                                ),
                            ),
                            'acceptlanguage',
                        ),
),
```

In both cases if I call an ajax request to http://www.site.com/events/index/savecoords I get a 404 error. 

Any hint? Is this a bug?
Regards

**Comment by basz** [comment]
Hi, maintenance of this package has been taken over by me. If you still feel this issue is valid please go ahead and create a PR or contribute in any other manner. It's appreciated.

**Comment by koseduhemak** [comment]
I have the same problem of rewritten asset paths... I think it is because I use an AssetManager (https://github.com/RWOverdijk/AssetManager). To insert a script or css file I call a ViewHelper `$this->asset('js/123.js');` which uses the URL ViewHelper (which I guess is getting overwritten or decorated by the SlmLocale module and therefore decorates the generated asset url with the language dependent stuff).

**Comment by koseduhemak** [comment]
Added pull request with new strategy to deal with asset management.

**Comment by svycka** [comment]
fixed and tagged as 0.3.0
